components/VoteList/VoteList.tsx

```diff
@@ -1,36 +1,41 @@
-import { tabletAndUnder, tabletMax } from "constant";
+import { tabletMax } from "constant";
 import { useWindowSize } from "hooks";
-import styled from "styled-components";
+import { ActivityStatusT } from "types";
+import { VoteListItem } from "./VoteListItem";
+import { VoteTableHeadings } from "./VoteTableHeadings";
+import { VoteTableRow } from "./VoteTableRow";
+import { VoteListItemProps } from "./shared.types";
 
 interface Props {
-  headings: JSX.Element;
-  rows: JSX.Element[];
+  activityStatus: ActivityStatusT;
+  data: VoteListItemProps[];
 }
-export function VoteList({ headings, rows }: Props) {
+export function VoteList({ data, activityStatus }: Props) {
   const { width } = useWindowSize();
 
   if (!width) return null;
 
   const isTabletAndUnder = width <= tabletMax;
 
+  if (isTabletAndUnder)
```

Comment:
this is the most important change. we now have two different components for mobile and desktop. this allowed me to separate the styling for the table stuff from the mobile list stuff